src/theme/components/heading.ts

```diff
@@ -0,0 +1,105 @@
+// Style object for the Heading's "control" or visual icon.
+const baseStyle = {
```

Comment:
If these are empty, it's okay to not include them in the exported object.

Reply:
OK.  I was worried that removing them would break the data object.